examples/pages/examples/cellSpacing.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import Editor, {
+  Value,
+  AutoForm,
+  AutoFields,
+  makeUniformsSchema,
+} from '@react-page/editor';
+import slate from '@react-page/plugins-slate';
+import image from '@react-page/plugins-image';
+import customLayout from '../../plugins/customLayoutPluginWithCellSpacing';
+import PageLayout from '../../components/PageLayout';
+import { cellSpacingDemo } from '../../sampleContents/cellSpacing';
+
+// Define which plugins we want to use.
+const cellPlugins = [slate(), image, customLayout];
+
+interface CellSpacingState {
+  value?: Value;
+  readOnly?: boolean;
+  outlineEditor?: boolean;
+  outlineCells?: boolean;
+  cellSpacingX?: string;
+  cellSpacingY?: string;
+}
+
+// Remove samples stylesheet cell padding and optionally outline cell and editor boundaries.
+function styles(state: CellSpacingState) {
+  let styles = '.react-page-cell-inner-leaf { padding: 0; }';
+
+  if (state.outlineCells) {
+    styles +=
+      '.react-page-cell-inner:not(.react-page-row) > div { outline: 1px solid red; }';
+  }
+
+  if (state.outlineEditor) {
+    styles += '.editor { outline: 1px solid green; }';
+  }
+
+  return styles;
+}
+
+export default function CellSpacing() {
+  const [state, setState] = React.useState<CellSpacingState>({
+    value: cellSpacingDemo,
+    outlineEditor: true,
+    outlineCells: true,
+    cellSpacingX: '10',
+    cellSpacingY: '10',
+  });
+
+  const style = React.useMemo(
+    () => <style dangerouslySetInnerHTML={{ __html: styles(state) }} />,
+    [state.outlineCells, state.outlineEditor]
+  );
+
+  const schema = makeUniformsSchema({
+    type: 'object',
+    properties: {
+      outlineEditor: { type: 'boolean', title: 'Outline the Editor in Green' },
+      outlineCells: { type: 'boolean', title: 'Outline Cells in Red' },
+      readOnly: { type: 'boolean', title: 'Read Only' },
+      cellSpacingX: { type: 'string', title: 'Horizontal Cell Spacing' },
+      cellSpacingY: { type: 'string', title: 'Vertical Cell Spacing' },
```

Comment:
use type `integer` or `number` instead of strings

Reply:
@macrozone , I want them to be able to give us random strings. I'm checking how we can survive the stuff that will convert to NaNs.

Reply:
you mean em and rem and sort? that makes sense.

You can also check whether React.CssProperties["paddingLeft"] or so has a more precise definition for paddings

Reply:
ah no, you do math with it (e.g. -cellSpacingX), so it has to be a number. see coment above